src/codemodder/__main__.py

```diff
@@ -192,6 +192,14 @@ def run(argv, original_args) -> int:
     return 0
 
 
-if __name__ == "__main__":
+def run_script():
+    # TODO: I'm not sure why this needs to be parsed out separately
+    # Maybe it has something to do with the invocation as python -m codemodder.
```

Comment:
IIRC it's to make testing easier, which isn't the best reason in the world. I do think we should go ahead and deprecate this and update the tests, but if you want to do that later that's cool with me.

Reply:
I will update the tests 👍